app/models/listing.rb

```diff
@@ -10,9 +10,16 @@ class Listing < ActiveRecord::Base
   has_many :listing_images
   has_many :reservations
   scope :city, -> (city) { where city: city }
-  scope :max_guests, -> (max_guests) { where("max_guests >= ?", max_guests) }
+  scope :max_guests, -> (max_guests) { where("max_guests >= ?", max_guests)
+    .order(max_guests: :asc) }
```

Comment:
Expression at 14, 30 should be on its own line.<br>Place the . on the previous line, together with the method call receiver.